modules/c++/re/source/RegexPCRE.cpp

```diff
@@ -20,39 +20,41 @@
  *
  */
 
-#if defined(USE_PCRE)
+#include "re/Regex.h" // this has to come before the #ifdef checks below
 
-#include "re/PCRE.h"
+#if !defined(__CODA_CPP11)
 
-re::PCRE::PCRE(const std::string& pattern, int flags) :
-    mPattern(pattern), mPCRE(NULL)
+#include <sstream>
+
+re::Regex::Regex(const std::string& pattern) :
+    mPattern(pattern), OVECCOUNT(999), mPCRE(NULL), mOvector(OVECCOUNT)
 {
     if (!mPattern.empty())
-        compile(mPattern, flags);
+    {
+        compile(mPattern);
+    }
 }
 
-void re::PCRE::destroy()
+void re::Regex::destroy()
 {
     if (mPCRE != NULL)
     {
         pcre_free(mPCRE);
     }
 }
 
-re::PCRE::~PCRE()
+re::Regex::~Regex()
 {
     destroy();
 }
 
-re::PCRE::PCRE(const re::PCRE& rhs)
+re::Regex::Regex(const re::Regex& rhs) :
+    mPattern(rhs.mPattern), OVECCOUNT(999), mPCRE(NULL), mOvector(OVECCOUNT)
```

Comment:
OK sorry one more name change here... if it's a static constant, then it should be all caps and with underscores between the words to make it really obvious that it's static (and the `m` no longer applies).  So, change to `OVECTOR_COUNT`.